fire/SFMainMod.F90

```diff
@@ -902,28 +911,55 @@ subroutine  crown_damage ( currentSite )
-                else
-                   ! Flames part of way up canopy. 
-                   ! Equation 17 in Thonicke et al. 2010. 
-                   ! flames over bottom of canopy but not over top.
-                   if ((currentCohort%hite > 0.0_r8).and.(currentPatch%SH >=  &
-                        (currentCohort%hite-currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft)))) then 
-
-                           currentCohort%fraction_crown_burned =  (currentPatch%SH-currentCohort%hite*(1.0_r8- &
-                                EDPftvarcon_inst%crown(currentCohort%pft)))/(currentCohort%hite* &
-                                EDPftvarcon_inst%crown(currentCohort%pft)) 
-
-                   else 
-                      ! Flames over top of canopy. 
-                      currentCohort%fraction_crown_burned =  1.0_r8 
-                   endif
+                
+                ! height_cbb = clear branch bole height at base of crown (m)
+                ! inst%crown = crown_depth_frac (PFT)
+                crown_depth                          = currentCohort%hite*EDPftvarcon_inst%crown(currentCohort%pft) 
+                height_cbb                           = currentCohort%hite - crown_depth
+                passive_crown_FI                     = 0.0_r8  
+                ignite_crown                         = 0.0_r8  
+                currentCohort%passive_crown_fire_flg = 0
 
-                endif
+                
+                ! Evaluate for passive crown fire ignition
+                if (EDPftvarcon_inst%crown_fire(currentCohort%pft) == 1) then
+                   
+                   ! Note: crown_ignition_energy to be calculated based on foliar moisture content from FATES-Hydro
+                   ! EQ 3 Van Wagner 1977
+                   ! h = crown_ignite_energy (kJ/kg), m = foliar moisture content based on dry fuel (%)
+                   ! crown_ignite_energy = 460 + 26 * m
+
+                   ! Crown fuel ignition potential, EQ 8 Bessie and Johnson 1995
+                   ! Van Wagner EQ4 FI = (Czh)**3/2 where z=crown base height,h=heat crown ignite energy, FI=fire intensity
+                   ! 0.01 = C from Van Wagner 1977 EQ4 for crown of base height 6m, 100% FMC, and FI 2500kW/m
+                   passive_crown_FI = (0.01_r8 * height_cbb *EDPftvarcon_inst%crown_ignite_energy(currentCohort%pft))**1.5_r8
+
+                   if (passive_crown_FI >= crown_fire_threshold) then ! 200 kW/m = threshold for crown fire potential
+                      
+                      ! Initiation of passive crown fire, EQ 14 Bessie and Johnson 1995
+                      ignite_crown = currentPatch%FI/passive_crown_FI
+                      
+                      if (ignite_crown >= 1.0_r8) then
+                         currentCohort%passive_crown_fire_flg = 1 ! passive crown fire ignited
+                         currentCohort%fraction_crown_burned =  1.0_r8
```

Comment:
@jkshuman we would change this line to:
```suggestion
                         currentCohort%fraction_crown_burned =  EDPftvarcon_inst%crown_fire(currentCohort%pft)
```
to account for pft vulnerability to passive crown fire, instead of setting to 1 for all pfts.

Reply:
@slevisconsulting sure. then users can decide how to implement